src/service_aggregator.py

```diff
@@ -159,10 +169,11 @@ async def entry(message, guid, coalesce_type, caller) -> (dict, int):
                 {"id": "filter_message_top_n", "parameters": {"max_results": 500}},
             ]
         elif pathfinder: 
+            # ignoring scoring for now, will have to work on this
             workflow_def = [
                 {"id": "lookup"},
-                {"id": "overlay_connect_knodes"},
-                {"id": "score"},
+                # {"id": "overlay_connect_knodes"},
+                # {"id": "score"},
```

Comment:
What's the plan here?

Reply:
Depends on O&O. Pathfinder may not have any ARA scoring. No solid plans yet.

Reply:
Ok, sounds good!